platform-sdk/consensus-gossip/src/main/java/module-info.java

```diff
@@ -1,3 +1,6 @@
 module com.hedera.service.gossip {
     exports com.hedera.service.gossip;
+
+    requires transitive com.swirlds.common;
```

Comment:
Is this for a new publish group? I didn't think we were publishing anything new to com.swirlds.*